UserApi/UserApi/Services/UserAccountService.cs

```diff
@@ -108,14 +103,17 @@ public async Task<User> GetUserByFilter(string filter)
                 };
             }
 
-            if (responseMessage.StatusCode == HttpStatusCode.NotFound) return null;
+            if (responseMessage.StatusCode == HttpStatusCode.NotFound)
```

Comment:
Can you reduce multiple if's by combining the checks ?

Reply:
There's a bit different logic in there so I'm not sure I can.